src/adapters/scene_category.js

```diff
@@ -19,12 +19,12 @@ export default class SceneCategory {
     listen('highlight_category_marker', (poi, highlight) => {
       this.highlightPoiMarker(poi, highlight);
     });
-    listen('click_category_poi', (poi, categoryName) => {
-      this.selectPoi(poi, categoryName);
+    listen('click_category_poi', (poi, resource) => {
```

Comment:
If we refactor the filters a lot of things will have to be rename to stop referring to 'category'. Like this event.
But I guess it's ok to wait for the next PR and keep this one focused on the bug fix.